cves/kernel/CVE-2014-4667.yml

```diff
@@ -116,359 +27,95 @@ upvotes_instructions: |
   upvotes score on your branch.
 upvotes:
```

Comment:
I think this is cool, it'd be interesting to learn about the syntax of this specially crafted SCTP packet. +3